courses/machine_learning/deepdive2/text_classification/labs/classify_text_with_bert.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>Clarify ordering of layers in model. Some other helpful hints to students might be reiterating you will be using the pooled output accessible using the dictionary syntax like above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1412/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>I added a list and clarified the pooled output. Added it as a hint. This was a really good suggestion since this is the core part of the lab.</p>